pkgs/servers/klipper/default.nix

```diff
@@ -15,6 +15,12 @@ stdenv.mkDerivation rec {
     sha256 = "puAkSGL0DD0JUWejPdzr7zKIW2UP2soBBtgm2msUKzA=";
   };
 
+  # We have no LTO on i686 since commit 22284b0
```

Comment:
Please use the long sha as in the future this might become ambiguous.

Reply:
Thanks, done.